Src/FluentAssertions/Execution/AssertionScope.cs

```diff
@@ -48,25 +50,23 @@ public DeferredReportable(Func<string> valueFunc)
     #endregion
 
     /// <summary>
-    /// Starts a named scope within which multiple assertions can be executed
+    /// Starts an unnamed scope within which multiple assertions can be executed
     /// and which will not throw until the scope is disposed.
     /// </summary>
-    public AssertionScope(string context)
-        : this()
+    public AssertionScope()
+        : this(new CollectingAssertionStrategy())
     {
-        if (!string.IsNullOrEmpty(context))
-        {
-            Context = new Lazy<string>(() => context);
-        }
+        SetCurrentAssertionScope(this);
```

Comment:
`SetCurrentAssertionScope` is already called by the chained constructor call.

Reply:
Good catch (albeit harmless).